Makefile

```diff
@@ -8,7 +8,7 @@ EXEC_NAME := $(OUTPUT_DIR)/protoc-gen-ext
 SOURCES := $(shell find . -name "*.go" | grep -v test.go)
 VERSION ?= $(shell git describe --tags)
 
-GO_BUILD_FLAGS := GO111MODULE=on CGO_ENABLED=0 GOARCH=amd64
```

Comment:
Will this impact any of the work dealing with CGO for GraphQL?

Reply:
I don't believe so. 
These build flags would be just for building protoc-gen-ext binary, so even if the binary is used downstream, it doesn't preclude us from using the cgo specific build flag then, I believe.